user-storage-jpa/README.md

```diff
@@ -21,45 +21,27 @@ runtime. Before using this example, you should probably read the User Storage SP
 System Requirements
 -------------------
 
-You need to have <span>Keycloak</span> running.
-
-All you need to build this project is Java 8.0 (Java SDK 1.8) or later and Maven 3.3.3 or later.
-
+All you need to build this project is Java 11 (Java SDK 11) or later and Maven 3.3.3 or later.
 
 Build and Deploy the Quickstart
 -------------------------------
 
-You must first deploy the datasource it uses.
-Start up the <span>Keycloak</span> server.  Then in the directory of this example type the following maven command:
-
-   ````
-   mvn -Padd-datasource install
-   ````
+You must first configure the datasource it uses. 
+For that, copy the [conf/quarkus.properties](conf/quarkus.properties) to the `conf` directory of the server distribution.
+In this file, you have a named datasource using the same name as the [src/resources/META-INF/persistence.xml](persistence unit) from the quickstart. By using the same name,
```

Comment:
The linked resource is a) in the wrong order and b) there's a path missing
It should be: `[persistence unit](src/main/resources/META-INF/persistence.xml)`

Reply:
Done